### PR TITLE
Add docker development configuration.

### PR DESCRIPTION
**Pull request by dsummersl**

### CODE REVIEW

**Comment by jszwedko** [comment]
This is great thank you!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM ruby:2.2.3
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```
Comment (jszwedko) [inline Dockerfile]: Ah, I did just notice `nodejs` in here. Is this something your fork requires, perhaps?

Comment (dsummersl) [inline Dockerfile]: I added it b/c execjs causes `Could not find a JavaScript runtime` errors without some runtime. Not exactly sure what is going on; the rails_admin/coffee-script or uglifier packages are require execjs.

Comment (dsummersl) [inline Dockerfile]: (nothing we added to our branch, to answer your question)

Comment (jszwedko) [inline Dockerfile]: Aha! Gotcha, I can believe that.

**Pull request merged**